nixos/modules/services/mail/rss2email.nix

```diff
@@ -0,0 +1,226 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.rss2email;
+  conf = pkgs.writeText "rss2email.cfg" (lib.generators.toINI {} ({
+      DEFAULT = default-config // cfg.configuration // { to = cfg.to; };
```

Comment:
Along with the other changes, you can then set the `to` value via `services.rss2email.config.to = cfg.to` in the `config` section of this module.